### PR TITLE
Data validation tests for multiple files in parallel

### PR DESCRIPTION
**Pull request by souravgupta-msft**

### CODE REVIEW

File: test/e2e_tests/data_validation_test.go [hunk 1]
```diff
@@ -181,6 +197,77 @@ func (suite *dataValidationTestSuite) TestFileUpdate() {
...
+
+	// create the file in local directory
+	srcFile, err := os.OpenFile(localFilePath, os.O_CREATE, 0777)
+	suite.Equal(nil, err)
+	srcFile.Close()
+
+	// write to file in the local directory
+	var fileBuff []byte
+	if fileSize == "large" {
+		fileBuff = make([]byte, (500 * 1024 * 1024))
```
Comment (vibhansa-msft) [inline test/e2e_tests/data_validation_test.go:215]: can we add some test case for 2-3 files which are 1/2gb in size.

**Review by vibhansa-msft — APPROVED** [review]

File: test/e2e_tests/data_validation_test.go (hunk 1 above)
Comment (souravgupta-msft) [reply test/e2e_tests/data_validation_test.go:217]: Done

**Review by vibhansa-msft — APPROVED** [review]

**Comment by gapra-msft** [comment]
Looks like a bunch of validation tests aren't passing. Do we know why this is happening?

**Comment by souravgupta-msft** [comment]
> Looks like a bunch of validation tests aren't passing. Do we know why this is happening?

There is one test case failing for validating multiple 1 GB files. Will look into it.

**Review by gapra-msft — APPROVED** [review]

**Review by tasherif-msft — APPROVED** [review]

**Review by vibhansa-msft — APPROVED** [review]

**Review by gapra-msft — APPROVED** [review]

**Pull request merged**